pkg/spec/createconfig.go

```diff
@@ -293,6 +295,39 @@ func (c *CreateConfig) GetVolumeMounts(specMounts []spec.Mount) ([]spec.Mount, e
 	return m, nil
 }
 
+// GetVolumesFrom reads the create-config artifact of the container to get volumes from
+// and adds it to c.Volumes of the curent container.
+func (c *CreateConfig) GetVolumesFrom() error {
+	var options string
+	for _, vol := range c.VolumesFrom {
+		splitVol := strings.Split(vol, ":")
+		if len(splitVol) == 2 {
+			options = splitVol[1]
+		}
+		ctr, err := c.Runtime.LookupContainer(splitVol[0])
+		if err != nil {
+			return errors.Wrapf(err, "error looking up container %q", splitVol[0])
+		}
+		var createArtifact CreateConfig
+		artifact, err := ctr.GetArtifact("create-config")
+		if err != nil {
+			return errors.Wrapf(err, "error geting create-config artifact for %q", splitVol[0])
```

Comment:
Typo: geting -> getting

Reply:
Also, probably better to use ctr.ID() here instead of splitVol[0]

Reply:
So the synatx is `--volumes-from my-ctr:ro,z` and I need to look up the volumes `my-ctr` has bind mounted in so I can do the same for the current container being created. Hence using splitVol[0].